<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="freqLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="MeterBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Meters</string>
    </property>
    <widget class="QwtThermo" name="power">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>461</width>
       <height>51</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="alarmLevel">
      <double>90.000000000000000</double>
     </property>
    </widget>
    <widget class="QwtThermo" name="swr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>451</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="upperBound">
      <double>5.000000000000000</double>
     </property>
     <property name="scaleMaxMajor">
      <number>5</number>
     </property>
     <property name="scaleMaxMinor">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="scalePosition">
      <enum>QwtThermo::TrailingScale</enum>
     </property>
     <property name="alarmEnabled">
      <bool>true</bool>
     </property>
     <property name="alarmLevel">
      <double>3.000000000000000</double>
     </property>
     <property name="origin">
      <double>0.000000000000000</double>
     </property>
     <property name="value">
      <double>1.500000000000000</double>
     </property>
    </widget>
    <widget class="QwtThermo" name="alc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>451</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="upperBound">
      <double>5.000000000000000</double>
     </property>
     <property name="scaleMaxMajor">
      <number>5</number>
     </property>
     <property name="scaleMaxMinor">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="scalePosition">
      <enum>QwtThermo::TrailingScale</enum>
     </property>
     <property name="alarmEnabled">
      <bool>true</bool>
     </property>
     <property name="alarmLevel">
      <double>3.000000000000000</double>
     </property>
     <property name="value">
      <double>1.500000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>SWR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ALC</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Power (W)</string>
     </property>
    </widget>
    <widget class="QwtThermo" name="idd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>451</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="upperBound">
      <double>25.000000000000000</double>
     </property>
     <property name="scaleMaxMajor">
      <number>5</number>
     </property>
     <property name="scaleMaxMinor">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="scalePosition">
      <enum>QwtThermo::TrailingScale</enum>
     </property>
     <property name="alarmEnabled">
      <bool>true</bool>
     </property>
     <property name="alarmLevel">
      <double>20.000000000000000</double>
     </property>
     <property name="origin">
      <double>0.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Idd (Finals Current) Amps</string>
     </property>
    </widget>
   </widget>
   <widget class="QwtWheel" name="Wheel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>64</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SWR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>160</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ALC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>190</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>220</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Idd</string>
    </property>
   </widget>
   <widget class="QLabel" name="poText">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>190</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="swrText">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sText">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="iddText">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>220</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="alcText">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>160</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QwtThermo" name="sMeter">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>60</width>
      <height>250</height>
     </rect>
    </property>
    <property name="upperBound">
     <double>255.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>60</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionStatus"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
    <addaction name="actionUpdate_Frequency"/>
   </widget>
   <widget class="QMenu" name="menuQuick_Action">
    <property name="title">
     <string>Quick Action</string>
    </property>
    <addaction name="actionDNR_SSB"/>
    <addaction name="actionDNR_CW"/>
    <addaction name="actionFT8"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuQuick_Action"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="toolTip">
    <string extracomment="Hello Status"/>
   </property>
  </widget>
  <action name="actionFIle">
   <property name="text">
    <string>FIle</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="text">
    <string>Status</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionDNR_SSB">
   <property name="text">
    <string>DNR SSB</string>
   </property>
  </action>
  <action name="actionDNR_CW">
   <property name="text">
    <string>DNR CW</string>
   </property>
  </action>
  <action name="actionFT8">
   <property name="text">
    <string>FT8</string>
   </property>
  </action>
  <action name="actionUpdate_Frequency">
   <property name="text">
    <string>Update Frequency</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtThermo</class>
   <extends>QWidget</extends>
   <header>qwt_thermo.h</header>
  </customwidget>
  <customwidget>
   <class>QwtWheel</class>
   <extends>QWidget</extends>
   <header>qwt_wheel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
